test/token/ERC20/ERC20.test.js

```diff
@@ -287,6 +287,46 @@ contract('ERC20', function (accounts) {
     });
   });
 
+  describe('_update', function () {
+    const amount = initialSupply + 1;
```

Comment:
This seems simpler. I don't see a reason to involve `initialSupply`.

```suggestion
    const amount = 1;
    
```